### PR TITLE
chore: removing VE from cards

### PR DESCRIPTION
**Pull request by JackShort**
* this PR makes no logic changes or stylistic changes only removes VE in favor of styled components

### CODE REVIEW

File: src/nft/components/collection/Card.tsx [hunk 1]
```diff
@@ -277,13 +297,46 @@ function getHeightFromAspectRatio(uniformAspectRatio: UniformAspectRatio, render
...
+  uniformAspectRatio?: UniformAspectRatio
+  setUniformAspectRatio?: (uniformAspectRatio: UniformAspectRatio) => void
+  renderedHeight?: number
+  hovered: boolean
+  imageLoading: boolean
+  $hidden?: boolean
+}>`
+  width: 100%;
+  aspect-ratio: ${({ uniformAspectRatio, setUniformAspectRatio }) =>
+    `${getMediaAspectRatio(uniformAspectRatio, setUniformAspectRatio)}`};
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:330]: nit: I would calculate the aspect ratio in the component and then pass it into the image so not have any calculations in the css itself.

File: src/nft/components/collection/Card.tsx [hunk 2]
```diff
@@ -166,6 +163,32 @@ const StyledImageContainer = styled.div<{ isDisabled?: boolean }>`
...
+  position: relative;
+  border-radius: ${BORDER_RADIUS}px;
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  overflow: hidden;
+  padding-bottom: 12px;
+  border-radius: 16px;
+  box-shadow: rgba(0, 0, 0, 10%) 0px 4px 12px;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  transition: ${({ theme }) => `${theme.transition.duration.medium}`};
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:176]: nit: I don't think this is having any effect here. Locally I was able to remove it and didn't notice any change

File: src/nft/components/collection/Card.tsx [hunk 3]
```diff
@@ -300,40 +353,55 @@ const Image = ({
...
+  transform: translateY(-56px);
+`
+
+const StyledVideo = styled.video<{
+  uniformAspectRatio?: UniformAspectRatio
+  setUniformAspectRatio?: (uniformAspectRatio: UniformAspectRatio) => void
+  renderedHeight?: number
+}>`
+  width: 100%;
+  aspect-ratio: ${({ uniformAspectRatio, setUniformAspectRatio }) =>
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:395]: nit: same here with not doing manipulations in the css

File: src/nft/components/collection/Card.tsx [hunk 4]
```diff
@@ -360,31 +428,27 @@ const Video = ({
...
-          className={clsx(hovered && !isMobile && styles.cardImageHover, !imageLoaded && styles.loadingBackground)}
+          $hidden={shouldPlay}
         />
-      </Box>
+      </StyledMediaContainer>
       {shouldPlay ? (
         <>
-          <Box className={styles.playbackSwitch}>
+          <PlaybackButton>
             <PauseButtonIcon
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:452]: nit: Can we convert the pause and play icons to react feather?

File: src/nft/components/collection/Card.tsx [hunk 5]
```diff
@@ -537,43 +593,56 @@ interface CardDetailsContainerProps {
...
-  flex: 1;
 `
 
 interface ProfileNftDetailsProps {
   asset: WalletAsset
   hideDetails: boolean
 }
 
+const ProfileAssetNameRow = styled(Row)`
+  justify-items: flex-start; ;
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:631]: extra semi colon `;`

File: src/nft/components/collection/Card.tsx [hunk 6]
```diff
@@ -537,43 +593,56 @@ interface CardDetailsContainerProps {
...
+`
+
+const PrimaryRowContainer = styled.div`
+  overflow: hidden;
+  width: 100%;
+  flex-wrap: nowrap; ;
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:637]: extra semi colon `;`

File: src/nft/components/collection/Card.tsx (hunk 5 above)
Comment (cbachmeier) [reply src/nft/components/collection/Card.tsx:631]: Also I believe Rows are flex-start by default

File: src/nft/components/collection/Card.tsx [hunk 7]
```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
...
-        >
-          {assetName()}
-        </TruncatedTextRow>
+      <ProfileAssetNameRow>
+        <TruncatedTextRow color="textPrimary">{assetName()}</TruncatedTextRow>
         {asset.susFlag && <Suspicious />}
-      </Row>
-      <TruncatedTextRow className={buttonTextMedium} style={{ color: themeVars.colors.textPrimary }}>
+      </ProfileAssetNameRow>
+      <FloorPriceRow>
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:669]: You could remove FloorPriceRow and use `ThemedText.Subheader` with a lineheight of 20px

File: src/nft/components/collection/Card.tsx [hunk 8]
```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
...
 
-const SecondaryRow = ({ children }: { children: ReactNode }) => (
-  <Row height="20" justifyContent="space-between" marginTop="6">
-    {children}
-  </Row>
-)
+const StyledSecondaryRow = styled(Row)`
+  height: 20px;
+  justify-content: space-between;
+  margin-top: 6px; ;
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:711]: extra semi colon `;`

File: src/nft/components/collection/Card.tsx [hunk 9]
```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
...
-  </Row>
+  <StyledSecondaryDetails>{children}</StyledSecondaryDetails>
 )
 
+const SecondaryInfoContainer = styled.div`
+  color: ${({ theme }) => theme.textPrimary};
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  line-height: 20px;
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:730]: The secondary line seems to have a different line height than what's currently in prod. Here's a gif comparing the two
profileLineHeight

File: src/nft/components/collection/Card.tsx [hunk 10]
```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
...
-      className={subhead}
-    >
-      {children}
-    </Box>
-  )
+  return <SecondaryInfoContainer>{children}</SecondaryInfoContainer>
 }
 
+const TertiaryInfoContainer = styled.div`
+  color: ${({ theme }) => theme.textSecondary};
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:738]: Tertiary styles, notably fontWeight, seem different here compared to Prod
fontWeightTertiary

File: src/nft/components/collection/Card.tsx [hunk 11]
```diff
@@ -700,15 +775,18 @@ const Erc1155Controls = ({ quantity }: Erc1155ControlsInterface) => {
...
 const MarketplaceIcon = ({ marketplace }: { marketplace: string }) => {
-  return (
-    <Box
-      as="img"
-      alt={marketplace}
-      src={`/nft/svgs/marketplaces/${marketplace}.svg`}
-      className={styles.marketplaceIcon}
-    />
-  )
+  return <StyledMarketplaceIcon as="img" alt={marketplace} src={`/nft/svgs/marketplaces/${marketplace}.svg`} />
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:789]: can remove `as="img"`

File: src/nft/components/collection/Card.tsx [hunk 12]
```diff
@@ -744,44 +844,49 @@ const Ranking = ({ rarity, provider, rarityVerified, rarityLogo }: RankingProps)
...
+              <Row>{rarityVerified ? <RarityVerifiedIcon /> : null}</Row>
+            </RarityInfo>
           </MouseoverTooltip>
         </RankingContainer>
       )}
     </>
   )
 }
+
 const SUSPICIOUS_TEXT = 'Blocked on OpenSea'
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:873]: Would be a good time to add Translations

File: src/nft/components/collection/Card.tsx [hunk 13]
```diff
@@ -790,44 +895,46 @@ const Pool = () => {
   return (
     <MouseoverTooltip
       text={
-        <Box className={bodySmall}>
+        <ThemedText.BodySmall>
           This NFT is part of a liquidity pool. Buying this will increase the price of the remaining pooled NFTs.
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:899]: I would say add Translations but this is going to be removed so can be ignored.

File: src/nft/components/collection/Card.tsx [hunk 14]
```diff
@@ -790,44 +895,46 @@ const Pool = () => {
...
-        top="1/2"
-        style={{ transform: 'translate3d(-50%, -50%, 0)' }}
-        color="gray500"
-        className={body}
-      >
+    <NoContentContainerBackground height={height}>
+      <NoContentText>
         Content not
         <br />
         available yet
```
Comment (cbachmeier) [inline src/nft/components/collection/Card.tsx:935]: Translations

**Review by cbachmeier — COMMENTED** [review]
All small nits with a couple minor differences

File: src/nft/components/collection/Card.tsx (hunk 7 above)
Comment (JackShort) [reply src/nft/components/collection/Card.tsx:655]: it actually extends another styled component

**Pull request merged**